source/draw/gpu/opengl/OpenGLContext.ooc

```diff
@@ -146,11 +149,19 @@ OpenGLContext: class extends GpuContext {
 		}
 	}
 	update: func { this _backend swapBuffers() }
-	packToRgba: func (source: GpuImage, target: GpuImage, viewport: IntBox2D) {
+	packToRgba: func (source: GpuImage, target: GpuImage, viewport: IntBox2D, padding := 0) {
 		channels := 1
```

Comment:
Should be possible, I think it was that way before but I don't remember why it was removed.